server/graphql/v2/object/Update.js

```diff
@@ -0,0 +1,118 @@
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import { stripHTML } from '../../../lib/sanitize-html';
+import { DateString } from '../../v1/types';
+import { getIdEncodeResolver, IDENTIFIER_TYPES } from '../identifiers';
+import { Account } from '../interface/Account';
+
+import { Tier } from './Tier';
+
+const UpdateAudienceTypeEnum = new GraphQLEnumType({
+  name: 'UpdateAudienceTypeEnum',
+  description: 'Defines targets for an update',
+  values: {
+    ALL: {
+      description: 'Will be sent to collective admins and financial contributors',
+    },
+    COLLECTIVE_ADMINS: {
+      description: 'Will be sent to collective admins',
+    },
+    FINANCIAL_CONTRIBUTORS: {
+      description: 'Will be sent to financial contributors',
+    },
+  },
+});
+
+const Update = new GraphQLObjectType({
+  name: 'Update',
+  description: 'This represents an Update',
+  fields: () => {
+    return {
+      id: {
+        type: new GraphQLNonNull(GraphQLString),
+        resolve: getIdEncodeResolver(IDENTIFIER_TYPES.UPDATE),
+      },
+      legacyId: { type: GraphQLInt },
+      slug: { type: new GraphQLNonNull(GraphQLString) },
+      userCanSeeUpdate: {
+        description: 'Indicates whether or not the user is allowed to see the content of this update',
+        type: new GraphQLNonNull(GraphQLBoolean),
+        resolve(update, _, req) {
+          if (!update.isPrivate) {
+            return true;
+          }
+          return req.remoteUser && req.remoteUser.canSeeUpdates(update.CollectiveId);
+        },
+      },
+      isPrivate: { type: new GraphQLNonNull(GraphQLBoolean) },
+      title: { type: new GraphQLNonNull(GraphQLString) },
+      createdAt: { type: new GraphQLNonNull(DateString) },
+      updatedAt: { type: new GraphQLNonNull(DateString) },
+      publishedAt: { type: DateString },
+      image: { type: GraphQLString },
```

Comment:
This field is never set in the DB, looks like a feature that was never fully implemented. Let's not expose it

```suggestion
```